apps/src/code-studio/levels/levelGroup.js

```diff
@@ -144,6 +146,21 @@ window.initLevelGroup = function (
     }
   }
 
+  // Replaces emoji in a string with the given string.
+  // Returns the updated string.
+  // Source: http://crocodillon.com/blog/parsing-emoji-unicode-in-javascript
+  function replaceEmoji(source, replace) {
```

Comment:
We're always replacing it with blankCharacter, so drop replace and just sub in blankCharacter